crates/mpz-common/src/executor/mt.rs

```diff
@@ -141,11 +119,16 @@ where
             .take()
             .expect("children were not left uninitialized");
 
-        if children.len() < 1 {
-            children.alloc(&self.mux, 1).await?;
+        if children.len() < 2 {
```

Comment:
So if `children.len() == 1`, then we would have 3 after successful allocation. But I assume this is ok?

Reply:
No, `alloc(2)` brings the total count to 2. If there already was 1 thread, it will allocate only 1 extra thread in order to bring the total to 2.